ntbs-service/DataMigration/NotificationImportService.cs

```diff
@@ -102,7 +102,7 @@ private List<ImportResult> ImportNotificationGroups(PerformContext context, stri
         }
         private async Task<ImportResult> ValidateAndImportNotificationGroupAsync(PerformContext context, string requestId, List<Notification> notifications)
         {
-            var patientName = notifications.FirstOrDefault().FullName;
+            var patientName = notifications.FirstOrDefault()?.FullName;

```

Comment:
Why do we need the `?` here? We are guaranteed to have at least one notification in the list, no? Perhaps this should just be `First()`?